### PR TITLE
Labor branch merge request

### PR DESCRIPTION
**Pull request by noramurakozy**
Details:
- Random reuse fixed
- -= operator invalid use fixed
- throw exception fixed

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
   private double FAILURE_RATE = 0.0; //NOSONAR
 
   private int torpedoCount = 0;
+private Random generator = SecureRandom.getInstanceStrong();
```
Comment (arpadfodor) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Great solution

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +31,18 @@ public TorpedoStore(int numberOfTorpedos){
...
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
     }
 
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
+
```
Comment (aaronfodor) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:40]: Random newline :D

**Review by aaronfodor — COMMENTED** [review]
I think your fix is okay.

**Review by aaronfodor — APPROVED** [review]
I think it is okay now.

**Pull request merged**